libs/forcetk.mobilesdk.js

```diff
@@ -566,7 +578,7 @@ if (forcetk.Client === undefined) {
      */
     forcetk.Client.prototype.upsert = function(objtype, externalIdField, externalId, fields, callback, error) {
         return this.ajax('/' + this.apiVersion + '/sobjects/' + objtype + '/' + externalIdField + '/' + externalId
-        + '?_HttpMethod=PATCH', callback, error, "POST", JSON.stringify(fields));
+                         , callback, error, "PATCH", JSON.stringify(fields));
```

Comment:
@wmathurin Shouldn't we keep this way of making `PATCH` requests? On `Android`, there is no inbuilt `PATCH` support, so we rely on this while making a `POST` request to convert it to `PATCH`. On `iOS`, we drop it and treat it as `PATCH` directly, since it's supported.